tests/10-issuer.js

```diff
@@ -108,19 +112,31 @@ describe('Issue Credential - Data Integrity', function() {
         const {result, error} = await issuer.post({json: body});
         shouldThrowInvalidInput({result, error});
       });
-      it('"credential.type" items MUST be strings', async function() {
+      it('"credential.type" items MUST be strings.', async function() {
         this.test.cell = {
           columnId: name,
           rowId: this.test.title
         };
         const body = createRequestBody({issuer});
-        const invalidCredentialTypes = [null, true, 4, []];
-        for(const invalidCredentialType of invalidCredentialTypes) {
-          body.credential.type = invalidCredentialType;
-          const {result, error} = await issuer.post({json: {...body}});
-          shouldThrowInvalidInput({result, error});
-        }
+        const {result, data: issuedVc, error} = await issuer.post({json: body});
+        shouldReturnResult({result, error});
+        should.exist(issuedVc, 'Expected result to have data.');
+        result.status.should.equal(201, 'Expected statusCode 201.');
+        shouldBeIssuedVc({issuedVc});
       });
+      for(const [title, invalidType] of invalidCredentialTypes) {
```

Comment:
This is actually a good point, I'm not sure how the reporter would handle this though. In the actual report we don't support nested `it` test statements. In other words we don't support assertions on a normative statement in the reporter's results right now. So I think this would require refactoring the reporter and the html so that there could be something like this:

- "credential.type" items MUST be strings
    -  "credential.type" items MUST NOT be null
    - "credential.type" items MUST NOT be booleans
    - "credential.type" items MUST NOT be symbols

I also think that method helps to address concerns @msporny had with these PRs which is that the statements in the report need to match the statements in the Specification. So good idea @JSAssassin, but it will require some work.

Reply:
@JSAssassin ok so if the above is what you want it can be implemented by using this: add a new property to `test.cell` such as `lemma = true` and then in the template here: https://github.com/digitalbazaar/mocha-w3c-interop-reporter/blob/main/templates/matrix.hbs you would need to look from `lemma` and if `lemma` indent the test name. not sure how that would look, but basically the html would be the hard part.